extensions/control-plane/provision/provision-bigquery/src/main/java/org/eclipse/edc/connector/provision/gcp/BigQueryProvisioner.java

```diff
@@ -38,7 +38,6 @@
 import static java.util.concurrent.CompletableFuture.completedFuture;
 
 public class BigQueryProvisioner implements Provisioner<BigQueryResourceDefinition, BigQueryProvisionedResource> {
```

Comment:
please note that currently the `bqFactory.createBigQuery` method is called with a `serviceAccountName` that could be not the actual one, because on how the code is structured, please first value the `serviceAccountName` according to the possibility that it could be ADC and after then call the `bqFactory`

Reply:
I found that the GcpConfiguration javadoc was not updated: so when ADC is used, the serviceAccountName is null, which will result in getting the default service account object. It seems that, in the current implementation, the one invocation of createBigQuery should work, since serviceAccountName is initialized at line 68:
`var serviceAccountName = getServiceAccountName(resourceDefinition);`
and then used at line 86:
`var bigQuery = bqFactory.createBigQuery(serviceAccountName);`
Is that ok?

Reply:
Done